### ISSUE TITLE
Crash: critical crash after the second restart of the game and trying to enter the server 1.19.2

### ISSUE THREAD

**Issue by papillo12** [devs-immortal/The-Aether#723]
### What were you doing when the crash occured? How can we replicate it? 

1. First, start the game and enter the server
2. Next, restart the game and log back into the server
3. Finally, the game crashes the second time you start the game and enter the server 

### Did it crash the client or server?

Client

### Mod Version

 paradise-lost-0.2.0-beta+1.19

### Fabric API Version

 fabric-api-0.66.0+1.19.2

### Crash Report

```Shell
---- Minecraft Crash Report ----
// Don't do that.

Time: 2022-11-14 11:41:16
Description: Unexpected error

java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1058.method_4578()" because "this.parent$this" is null
	at net.minecraft.class_1058$class_4728.method_24128(class_1058.java:1540)
	at net.minecraft.class_1058$class_5790.method_4622(class_1058.java:441)
	at net.minecraft.class_1059.method_4612(class_1059.java:263)
	at net.minecraft.class_1059.method_4622(class_1059.java:272)
	at net.minecraft.class_1060.method_4622(class_1060.java:160)
	at net.minecraft.class_310.method_1574(class_310.java:1803)
	at net.minecraft.class_310.method_1523(class_310.java:1135)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_1058$class_4728.method_24128(class_1058.java:1540)
	at net.minecraft.class_1058$class_5790.method_4622(class_1058.java:441)
	at net.minecraft.class_1059.method_4612(class_1059.java:263)
	at net.minecraft.class_1059.method_4622(class_1059.java:272)
	at net.minecraft.class_1060.method_4622(class_1060.java:160)
Mixins in Stacktrace: None found

-- Affected level --
Details:
	All players: 1 total; [class_746['gorgorito'/197, l='ClientLevel', x=44.86, y=112.00, z=39.71]]
	Chunk stats: 16384, 467
	Level dimension: minecraft:overworld
	Level spawn location: World: (1,63,2), Section: (at 1,15,2 in 0,3,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 183298 game time, 183298 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:453)
	at net.minecraft.class_310.method_1587(class_310.java:2341)
	at net.minecraft.class_310.method_1514(class_310.java:792)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Mixins in Stacktrace: None found

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, lambdabettergrass/default, Immersive Weathering Generated Pack, Snowyspirit Generated Pack

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 1386870504 bytes (1322 MiB) / 3309305856 bytes (3156 MiB) up to 8589934592 bytes (8192 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 2700X Eight-Core Processor         
	Identifier: AuthenticAMD Family 23 Model 8 Stepping 2
	Microarchitecture: unknown
	Frequency (GHz): 3.70
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: NVIDIA GeForce RTX 2080
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1e87
	Graphics card #0 versionInfo: DriverVersion=31.0.15.2686
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.00
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.00
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 31669.30
	Virtual memory used (MB): 20375.73
	Swap memory total (MB): 15360.00
	Swap memory used (MB): 421.54
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx8192m -Xms8192m -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms256m
	Fabric Mods: 
		advanced_runtime_resource_pack: Runtime Resource Pack 0.6.4
		advancementplaques: Advancement Plaques 1.4.6
		animatica: Animatica 0.5+1.19
		antighost: AntiGhost 1.19.1-fabric0.58.5-1.1.5
			crowdin-translate: CrowdinTranslate 1.4+1.19
		architectury: Architectury 6.3.49
		balm-fabric: Balm 4.5.2+0
		bambooeverything: Bamboo Everything 2.2.4-build.33+mc1.19.2
		bcc: BetterCompatibilityChecker 2.0.2-build.16+mc1.19.1
		betteranimationscollection: Better Animations Collection 4.0.4
		betterpingdisplay: Better Ping Display 1.1.1
		betterthirdperson: Better Third Person 1.8.1
		bettervillage: Better village 2.0.0
		biomemakeover: Biome Makeover 1.19.2-1.6.0
		blur: Blur (Fabric) 2.6.0
		bookshelf: Bookshelf 16.1.11
		bosses_of_mass_destruction: Bosses of Mass Destruction (Beta) 1.4.3-1.19.2
			maelstrom_library: Maelstrom Library 1.3-1.19-pre1
			multipart_entities: MultipartEntities 1.2-1.19-pre1
		bountiful: Bountiful 3.0.0
		bowinfinityfix: Bow Infinity Fix 2.5.0
		buildinggadgets: st'ructure tools 1.0.30
			mm: Manningham Mills 2.3
			team_reborn_energy: Energy 2.2.0
		byg: Oh The Biomes You'll Go 2.0.0.13
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
		capybara: Capybara 1.0.8
		cardinal-components: Cardinal Components API 5.0.2
			cardinal-components-base: Cardinal Components API (base) 5.0.2
			cardinal-components-block: Cardinal Components API (blocks) 5.0.2
			cardinal-components-chunk: Cardinal Components API (chunks) 5.0.2
			cardinal-components-entity: Cardinal Components API (entities) 5.0.2
			cardinal-components-item: Cardinal Components API (items) 5.0.2
			cardinal-components-level: Cardinal Components API (world saves) 5.0.2
			cardinal-components-scoreboard: Cardinal Components API (scoreboard) 5.0.2
			cardinal-components-world: Cardinal Components API (worlds) 5.0.2
		carrier: Carrier 1.11.0
		citresewn: CIT Resewn 1.1.2+1.19.2
			citresewn-defaults: CIT Resewn: Defaults 1.1.2+1.19.2
		cloth-api: Cloth API 4.0.65
			cloth-basic-math: cloth-basic-math 0.6.1
			cloth-client-events-v0: Cloth Client Events v0 4.0.65
			cloth-common-events-v1: Cloth Common Events v1 4.0.65
			cloth-scissors-api-v1: Cloth Scissors API v1 4.0.65
			cloth-utils-v1: Cloth Utils v1 4.0.65
		cloth-config: Cloth Config v8 8.2.88
		collective-fabric: Collective (Fabric) 5.15
			fiber: fiber 0.23.0-2
		continuity: Continuity 2.0.1+1.19
		controlling: Controlling For Fabric 10.0+7
		cull-less-leaves: Cull Less Leaves 1.0.6
			com_github_llamalad7_mixinextras: mixinextras 0.0.12
			conditional-mixin: conditional mixin 0.3.0
		cullclouds: Cull Clouds 0.1.0
		dankstorage: Dank Storage 1.19.2-4.3
		dark-loading-screen: Dark Loading Screen 1.6.12
		dashloader: DashLoader 4.1.3+1.19
			com_github_luben_zstd-jni: zstd-jni 1.5.2-2
			dev_quantumfusion_hyphen: Hyphen 0.2.2
			dev_quantumfusion_taski: Taski 1.1.0
		deeperdarker: Deeper and Darker 1.1.4
			customportalapi: Custom Portal Api 0.0.1-beta54-1.19
			paragon: Paragon 1.0.3
		deepslatecutting: Deepslate Cutting 1.5.0
			maybe-data: Maybe data 1.3.2-1.19.2
		droplight: Droplight 1.0.5
		dummmmmmy: MmmMmmMmmMmm 1.19-1.5.3
		effective: Effective 1.4
		elytraslot: Elytra Slot 6.0.0+1.19.2
		emotecraft: Emotecraft 2.2.6-SNAPSHOT-build.44
			bendy-lib: Bendy api 2.1.3-test2
			player-animator: Player Animator 0.3.5
		enhancedblockentities: Enhanced Block Entities 0.7.2+1.19.2
			spruceui: SpruceUI 4.0.0+1.19
		entityculling: EntityCulling-Fabric 1.5.2-mc1.19
			com_logisticscraft_occlusionculling: occlusionculling 0.0.6-SNAPSHOT
		equipmentcompare: Equipment Compare 1.3.1
		exordium: Exordium 1.0.1-mc1.19.2
		expandeddelight: Expanded Delight 0.2.5
		fabric-api: Fabric API 0.66.0+1.19.2
			fabric-api-base: Fabric API Base 0.4.12+93d8cb8290
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.10+93d8cb8290
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.18+c6af733c90
			fabric-block-api-v1: Fabric Block API (v1) 1.0.0+12bfe4ea90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.21+c6af733c90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.2+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.12+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.1.8+93d8cb8290
			fabric-commands-v0: Fabric Commands (v0) 0.2.29+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.35+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.4.1+6064db2a90
			fabric-convention-tags-v1: Fabric Convention Tags 1.1.2+93d8cb8290
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.6+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.2.3+b598f4ac90
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.32+0dd10df690
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.0+a1d87cb890
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.29+c6af733c90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.29+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.22+c6af733c90
			fabric-item-api-v1: Fabric Item API (v1) 1.5.8+93d8cb8290
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.33+70efcffe90
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.22+aaaf9d3390
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.20+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.0+33ffe9ec90
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.7+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.10+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.4+93d8cb8290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.18+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.18+c6af733c90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.5+c6af733c90
			fabric-networking-v0: Fabric Networking (v0) 0.3.22+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.1.0+06e0e25190
			fabric-particles-v1: Fabric Particles (v1) 1.0.11+79adfe0a90
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.26+c6af733c90
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.13+12bfe4ea90
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.15+aeb40ebe90
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.21+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.16+12bfe4ea90
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.8+c6af733c90
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.23+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.11.0+73145abb90
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.1.0+aae9039d90
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.7.3+3e1ba48e90
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.27+93d8cb8290
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.1+1cc24b1b90
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.0+c4f28df590
			fabric-textures-v0: Fabric Textures (v0) 1.0.21+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.1+93d8cb8290
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.1+42d99c3290
		fabric-furnaces: Fabric Furnaces 2.2.0-1.19.2
			omega-config: OmegaConfig 1.2.3-1.18.1
		fabric-language-kotlin: Fabric Language Kotlin 1.8.6+kotlin.1.7.21
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.21
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.21
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.5
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.4.1
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.4.1
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.4.1
		fabricloader: Fabric Loader 0.14.10
		farmersdelight: Farmer's Delight 1.19.2-1.3.5
		farsight: Farsight Mod 1.19-2.1
		ferritecore: FerriteCore 4.2.1
		forgeconfigapiport: Forge Config API Port 4.2.6
		frame: Frame 0.26.1+1.19-fabric
			frame-api-base: Frame Base 0.2.1+906ebc04a9
			frame-blocks-v0: Frame Blocks (v0) 0.1.3+233e7428a9
			frame-config-v0: Frame Config (v0) 0.3.1+0fa45150a9
			frame-content-registries-v0: Frame Content Registries (v0) 0.4.0+2876b511a9
			frame-enchantments-v0: Frame Enchantments (v0) 0.1.0+4093105da9
			frame-game-rules-v0: Frame Game Rules (v0) 0.2.4+e7ce56a6a9
			frame-items-v0: Frame Items (v0) 0.1.3+906ebc04a9
			frame-loot-tables-v0: Frame Loot Tables (v0) 0.1.4+4f7341eca9
			frame-registries-v0: Frame Registries (v0) 0.1.1+0e0ec74ea9
			frame-rendering-v0: Frame Rendering (v0) 0.1.5+906ebc04a9
			frame-tabbed-item-groups-v0: Frame Tabbed Item Groups (v0) 0.2.4+0aca9b56a9
			frame-toymaker-v0: Frame Data Generation Helpers (v0) 0.6.1+b13741c3a9
			frame-woods-v0: Frame Woods (v0) 0.2.1+6a33fbada9
		frozenup: Frozen Up 2.0.0-fabric
		ftbchunks: FTB Chunks 1902.3.9-build.168
		ftblibrary: FTB Library 1902.3.10-build.156
		ftbteams: FTB Teams 1902.2.9-build.61
		fwaystones: Fabric Waystones 3.0.5+mc1.19.2
		geckolib3: Geckolib 3.1.35
			com_eliotlash_mclib_mclib: mclib 19
		go-fish: Go Fish 1.6.0-1.19.1
		goblintraders: Goblin Traders 1.5.2
		goodall: Goodall 1.1.1
		guardvillagers: Guard Villagers Fabric 1.19.2-1.0.16
		harvestwithease: Harvest with ease 4.0.0.2
		healingcampfire-fabric: Healing Campfire (Fabric) 4.0
		iammusicplayer: Iam Music Player 3.19.2
		iceberg: Iceberg 1.0.46
		illagerslovearmor: IllagersLoveArmor 1.0.2+1.19
		illuminations: Illuminations 1.10.11
		image2map: Image2Map 0.3.0
			map-canvas-api: Map Canvas API 0.2.0+1.19.1
			sgui: sgui 1.1.5+1.19.1
		immediatelyfast: ImmediatelyFast 1.0.4
		immersive_armors: Immersive Armors 1.4.2
		immersive_weathering: Immersive Weathering 1.19.2-1.2.2
		indium: Indium 1.0.9+mc1.19.2
		infinitybuttons: Infinity Buttons 2.2.1-mc1.19
		inventoryprofilesnext: Inventory Profiles Next 1.8.5
		iris: Iris 1.4.0
			io_github_douira_glsl-transformer: glsl-transformer 1.0.0-pre21.2
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.10.1
		ironchests: Iron Chests: Restocked 3.0.2
		itemborders: Item Borders 1.1.6
		jade: Jade 8.2.3
		java: OpenJDK 64-Bit Server VM 17
		jmi: JourneyMapIntegration 0.13-30
		journeymap: Journeymap 5.9.0beta2
			journeymap-api-fabric: JourneyMap API 1.19.1-1.9-fabric-SNAPSHOT
		kambrik: Kambrik 4.0-1.19.2
		lambdabettergrass: LambdaBetterGrass 1.3.0+1.19
		lambdynlights: LambDynamicLights 2.1.2+1.19
			pride: Pride Lib 1.1.2+1.19
		languagereload: Language Reload 1.3.2
		lazydfu: LazyDFU 0.1.3
		libipn: libIPN 1.0.5
		libraryferret: Library ferret 3.0.0
		mcwbridges: Macaw's Bridges 2.0.5
		mcwdoors: Macaw's Doors 1.0.7
		mcwfences: Macaw's Fences and Walls 1.0.6
		mcwlights: Macaw's Lights and Lamps 1.0.4
		mcwpaintings: Macaw's Paintings 1.0.4
		mcwpaths: Macaw's Paths and Pavings 1.0.1
		mcwroofs: Macaw's Roofs 2.2.1
		mcwtrpdoors: Macaw's Trapdoors 1.0.7
		medieval_deco: PlayTics Deco 1.0
		medievalweapons: Medieval Weapons 1.4.1
			reach-entity-attributes: Reach Entity Attributes 2.3.0
		memoryleakfix: memory Leak Fix 1.19.1-0.7.0
		merchantmarkers: Merchant Markers 1.2.3
		midnightlib: MidnightLib 1.0.0
		minecraft: Minecraft 1.19.2
		minelib: mine11lib 0.1.3+1.19
			com_sksamuel_scrimage_scrimage-core: scrimage-core 4.0.31
		mixintrace: MixinTrace 1.1.1+1.17
		mo_glass: Mo Glass 1.6.1-MC1.19.2
		mobscarecrow: Mob Scarecrow 2.2.1-1.19.1
		modelfix: Model Gap Fix 1.8
		modmenu: Mod Menu 4.1.0
		moonlight: Moonlight 1.19.2-2.1.7
		mousetweaks: Mouse Tweaks 2.22
		musketmod: Musket Mod 1.4.6
		naturalist: Naturalist 2.1.1
		naturescompass: Nature's Compass 1.19.2-2.0.4-fabric
		netherdepthsupgrade: Nether Depths Upgrade 1.1.0
		no_fog: No Fog 1.3.1+1.16.5-1.19.2
		nochatreports: No Chat Reports 1.19.2-v1.13.12
		norecipebook: NoRecipeBook 2.2+1.19
		notenoughanimations: NotEnoughAnimations 1.6.2
		otyacraftengine: Otyacraft Engine 3.1.0
		owo: oωo 0.8.5+1.19
			blue_endless_jankson: jankson 1.2.1
		paintings: Paintings++ 10.2.4.0
		paradise_lost: Paradise Lost 0.2.0-beta+1.19
			gud_recipe_confidence: Recipe Confidence 1.0.2
			incubus_core: Incubus Core 1.7.1
			moretags: More Tags 3.0.5
		patchouli: Patchouli 1.19.2-77-FABRIC
		phosphor: Phosphor 0.8.1
		prism: Prism 1.0.3
		puzzleslib: Puzzles Lib 4.3.12
		reeses-sodium-options: Reese's Sodium Options 1.4.7+mc1.19.2-build.59
		repurposed_structures: Repurposed Structures 6.1.1+1.19
		roughlyenoughitems: Roughly Enough Items 9.1.572
			error_notifier: Error Notifier 1.0.9
		roughlyenoughprofessions: Roughly Enough Professions 1.1.2
		roughlyenoughresources: Roughly Enough Resources 2.6.0
		satin: Satin 1.9.0
		skiptransitions: Skip Transitions 1.3.2+1.18.x
		smoothboot: Smooth Boot 1.19-1.7.1
		snowyspirit: Snowy Spirit 1.19.2-2.0.10
		sodium: Sodium 0.4.4+build.18
			org_joml_joml: joml 1.10.4
		sodium-extra: Sodium Extra 0.4.11+mc1.19.2-build.68
			caffeineconfig: CaffeineConfig 1.0.0+1.17
		styledchat: Styled Chat 2.0.2+1.19.2
			fabric-permissions-api-v0: fabric-permissions-api 0.2-SNAPSHOT
			placeholder-api: Placeholder API 2.0.0-pre.1+1.19.2
			player-data-api: Player Data API 0.2.2+1.19.2
			predicate-api: Predicate API 0.1.0+1.19.2
		stylisheffects: Stylish Effects 4.3.2
		takesapillage: Takes a Pillage 1.0.2
		terrablender: TerraBlender 2.0.1.129
		thonkutil: ThonkUtil 2.15.4+1.19
			thonkutil-base: ThonkUtil Base 1.13.2+4a8c408a57
			thonkutil-capes-v1: ThonkUtil Capes (v1) 1.4.2+3eb2749857
			thonkutil-coords-v1: ThonkUtil Coords (v1) 1.1.2+8ff533c957
			thonkutil-customization-v1: ThonkUtil Customization (v1) 1.1.2+8ff533c957
			thonkutil-legacy: ThonkUtil Legacy 1.1.2+5d4263f557
			thonkutil-modchecker-v1: ThonkUtil ModChecker (v1) 1.1.3+bd4b387957
			thonkutil-potions-v0: ThonkUtil Potions (v0) 1.5.2+8ff533c957
			thonkutil-titlescreen-v1: ThonkUtil TitleScreen (v1) 1.2.2+8ff533c957
			thonkutil-trades-v1: ThonkUtil Trades (v1) 1.2.2+8ff533c957
		tipsmod: Tips 8.0.23
		toolstats: ToolStats 12.0.2
		transparent: Transparent 5.1.2
		trinkets: Trinkets 3.4.0
		tumbleweed: Tumbleweed 0.5.1
		visualworkbench: Visual Workbench 4.2.0
		voicechat: Simple Voice Chat 1.19.2-2.3.13
		yet-another-config-lib: YetAnotherConfigLib 1.7.0
		z_mcwfencesbyg: Macaw's Fences - BYG 1.19.2-1.0
		zoomify: Zoomify 2.8.0
			dev_isxander_settxi_settxi-core: settxi-core 2.10.3
			dev_isxander_settxi_settxi-gui: settxi-gui 2.10.3
			dev_isxander_settxi_settxi-kotlinx-serialization: settxi-kotlinx-serialization 2.10.3
			settxi-gui-yacl: Settxi Gui (YetAnotherConfigLib) 2.10.3
	Loaded Shaderpack: (off)
	Launched Version: 1.19.2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce RTX 2080/PCIe/SSE2 GL version 3.2.0 NVIDIA 526.86, NVIDIA Corporation
	Window size: 854x480
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: Fabric Mods, lambdabettergrass/default
	Current Language: English (US)
	CPU: 16x AMD Ryzen 7 2700X Eight-Core Processor
```

### Additional Information

it took me 2 days to find out what caused the crashes. 

[crash-2022-11-14_08.58.37-client.txt](https://github.com/devs-immortal/Paradise-Lost/files/10005708/crash-2022-11-14_08.58.37-client.txt)
[crash-2022-11-13_22.20.49-client.txt](https://github.com/devs-immortal/Paradise-Lost/files/10005709/crash-2022-11-13_22.20.49-client.txt)
[crash-2022-11-13_22.13.05-client.txt](https://github.com/devs-immortal/Paradise-Lost/files/10005710/crash-2022-11-13_22.13.05-client.txt)
[crash-2022-11-14_11.41.16-client.txt](https://github.com/devs-immortal/Paradise-Lost/files/10005711/crash-2022-11-14_11.41.16-client.txt)

[latest.log](https://github.com/devs-immortal/Paradise-Lost/files/10005712/latest.log)

**Comment by FireSparkC** [comment]
我也遇到了差不多的问题，但是我是新创建的世界无法进入。不使用该mod创建的世界可以正常游玩，也可正常使用该mod内容

**Comment by MBatt1** [comment]
Does the problem persist with the latest 2.0.2 version of Paradise Lost?

**Comment by SolidSprite** [comment]
This crash is still happening with version 2.0.2 for 1.19.2

**Comment by MBatt1** [comment]
Can you post a crash report from 2.0.2?

**Comment by Gamerbolts** [comment]
Hello! I am running 2.0.2 Tried to enter the World and crashed on load. Here is the crashlog. Was going through my large modpack and testing sections. Strange thing is this does not crash when it is loaded in the entire pack. I am not sure if there are embedded mods in Paradise Lost? If so maybe another mod I have is updating them (I know nothing of coding so this could be totally off) But here is my crashlog and log. Thanks for the help!

[crash-2023-01-08_15.38.26-client.txt](https://github.com/devs-immortal/Paradise-Lost/files/10369560/crash-2023-01-08_15.38.26-client.txt)
[latest.log](https://github.com/devs-immortal/Paradise-Lost/files/10369561/latest.log)

**Comment by MBatt1** [comment]
Could you try removing dashloader? That seems to be part of the conflict, though I'm curious as to how it would work as the entire pack but not sections.

**Comment by Gamerbolts** [comment]
Trying now, will update

**Comment by Gamerbolts** [comment]
Ok removed dashloader and the world loaded up. That is a very strange issue but yeah looks like Dashloader and Paradise Lost have a conflict.

**Comment by Gamerbolts** [comment]
Ok I tested with just Fabric API, Paradise Lost and Dashloader. So the World loads just fine the first time. When you close out of Minecraft then load back up and try to create a new world that is where it crashes. Makes sense as I have only created one world for testing on my modpack. I know Dashloader caches files to let loading launch faster after the first boot. So I am not sure if it is doing something strange with Paradise Lost, but here is my crashlog and log with just those 3 mods

[crash-2023-01-08_16.06.02-client.txt](https://github.com/devs-immortal/Paradise-Lost/files/10369601/crash-2023-01-08_16.06.02-client.txt)
[latest.log](https://github.com/devs-immortal/Paradise-Lost/files/10369605/latest.log)
